target/linux/mvebu/cortexa9/base-files/lib/preinit/06_set_iface_mac

```diff
@@ -29,7 +29,7 @@ preinit_set_mac_address() {
 		ip link set dev eth0 address $mac 2>/dev/null
 		ip link set dev eth1 address $mac 2>/dev/null
 		;;
-	marvell,a385-db-ap|solidrun,clearfog*a1)
+	marvell,a385-db-ap)
```

Comment:
Why do you leave `a385-db-ap` here? Shouldn't the renaming be dropped altogether?

Reply:
Because I have never seen a "Marvell Armada 385 Access Point Development Board" and don't feel to comfortable changing a device I have never used.